### DISCUSSION TITLE
Understanding Split Usage

### DISCUSSION THREAD

**Discussion by besseddrest** [zen-browser/desktop#1317] [General]
Just want to make sure that I'm doing this efficiently, and understanding what options are available

I'm on MacOS and I've created shortcuts:

* Cmd+ 1 - Hybrid Split
* Cmd+2 - Horizontal Split
* Cmd+3 - Vertical Split
* Cmd+4 - Toggle Split off

From what I can tell:

* to use hybrid split - you need at a minimum 3 tabs selected. Without any selected tabs I can't enable this split; and it seems like it will automatically lay things out for you based on the order of the tabs in the sidebar
* though by default, horizontal or vertical will just split with the adjacent tab, but you also have the option to select specific tabs
* To return to a single page view, or switch to a different split, you toggle it off first, in my case Cmd+4, then enable the desired split

But I'm noticing now...

* that when you enable splits it may actually keep the order in memory - i notice that in some instances when I toggle off to return to a single page view, the split actually returns to a previous split I had enabled, regardless of what tabs are selected

So some questions:

* does the browser keep a history of the order of enabled splits, like in a stack?
* is there a way to identify or highlight (other than a slight difference in tab color) which tabs are using which splits?
* is there a way to assign a tab to a split,  (like select the tab, select where you want it positioned), or repositioning them, other than ordering them in the left sidebar

Loving this browser so far, thank you!

**Comment by Iimvalue** [comment]
is there a way to assign a tab to a split, (like select the tab, select where you want it positioned), or repositioning them, other than ordering them in the left sidebar?

I found a method: when you split two tabs, an icon appears in the URL bar (as shown in the picture). This icon allows you to enable a rearrange feature, letting you drag and drop the tabs to your preferred positions. Once you finish dragging, the rearrange feature automatically disables itself.

![image](https://github.com/user-attachments/assets/1ce91508-a710-470f-be57-b45fe99f18c2)
